.github/workflows/ci.yml

```diff
@@ -0,0 +1,88 @@
+name: CI 
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: 0 14 * * MON-FRI # Every weekday at 14:00 UTC
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      matrix:
+        go-version: [1.18.x, 1.19.x]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
+      with:
+        go-version: ${{ matrix.go-version }}
+    # Need terraform to format the examples
+    - name: Setup Terraform
+      uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 # v2.0.3
+      with:
+        terraform_version: '1.4.*'
+        terraform_wrapper: false
+    - name: Generate
+      run: make generate
+    - name: Confirm no diff
+      run: |
+        git diff --compact-summary --exit-code || \
+          (echo "*** Unexpected differences after code generation. Run 'make generate' and commit."; exit 1)
+    - name: Build
+      run: make build
+  test:
+    needs: build
+    name: 'Acceptance Tests (OS: ${{ matrix.os }} / TF: ${{ matrix.terraform }})'
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          # - macos-latest
+          # - windows-latest
+          - ubuntu-latest
+        terraform:
+          - '1.0.*'
+          - '1.1.*'
+          - '1.2.*'
+          - '1.3.*'
+          - '1.4.*'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
+      with:
+        go-version-file: 'go.mod'
+    - name: Setup Terraform ${{ matrix.terraform }}
+      uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 # v2.0.3
+      with:
+        terraform_version: ${{ matrix.terraform }}
+        terraform_wrapper: false
+    - name: Run acceptance tests
+      run: make testacc | tee test_result.txt
+      shell: bash
+      env:
+        ACCTEST_PARALLELISM: 10
+        BASTIONZERO_API_SECRET: ${{ secrets.ACCEPTANCE_TESTS_API_SECRET }}
+        BASTIONZERO_HOST: ${{ secrets.ACCEPTANCE_TESTS_BASTIONZERO_HOST }}
```

Comment:
Yes. I made it a repository variable so that we can refer to the same value in both workflows that need it

bc6ff2c

https://github.com/bastionzero/terraform-provider-bastionzero/settings/variables/actions

Reply:
Ok nice even better